nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,65 @@
+const express = require("express");
+const app = express();
+
+// import data here
+
+const meals = require("./data/meals");
+const reviews = require("./data/reviews.json");
+const reservations = require("./data/reservations.json");
+
+// this is where you will be adding your routes
+
+
+const random = (arr) => arr[Math.floor(Math.random() * arr.length)];
+
+const mealsWithReviews = meals.map((meal) => {
+  return {
+    id: meal.id,
+    maxNumberOfGuests: meal.maxNumberOfGuests,
+    title: meal.title,
+    description: meal.description,
+    time: 12 + meal.id,
+    address: "asd " + meal.id,
+    price: meal.price,
+    reviews: reviews.filter((review) => review.mealId === meal.id),
+    createdAt: meal.createdAt,
+  };
+});
```

Comment:
Very nice, that way you avoid writing the same code in each route :1st_place_medal:  
what's with the "asd" address though :) :)

Reply:
:) :) Yes, it doesn't make sense I put  a meaningful address...